_posts/2010-09-27-adding-support-for-bags.html

```diff
@@ -85,7 +85,7 @@
 <p><a href="/assets/image007_500x233.jpg"><img class="alignnone size-full wp-image-377" alt="image007_500x233" src="/assets/image007_500x233.jpg" width="500" height="233" /></a></p>
 <p><strong>Additional Rules</strong></p>
 <p><strong>Customizable feed mapping</strong></p>
-<p>The following rules define the customizable feed mappings rules for Bags and are in addition to those already defined for entries <a href="https://www.odata.org/documentation/odata-version-2-0/atom-format#CustomizingtheRepresentationofanEntry"> here</a>.</p>
+<p>The following rules define the customizable feed mappings rules for Bags and are in addition to those already defined for entries <a href="/documentation/odata-version-2-0/atom-format#CustomizingtheRepresentationofanEntry"> here</a>.</p>
 <ul>
 <li>Each instance in the bag must be represented as a child element of the element representing the bag as a whole and be named 'element' and MUST be defined in the data service namespace (<a href="http://schemas.microsoft.com/ado/2007/08/dataservices">http://schemas.microsoft.com/ado/2007/08/dataservices</a>).</li>
```

Comment:
Link doesn't work https://schemas.microsoft.com/ado/2007/08/dataservices

Reply:
@KenitoInc Fixing dead links that point to external resources was beyond the scope of this pull request. In any event, I didn't have a valid alternative I could replace them with.